tests/__init__.py

```diff
@@ -375,6 +375,10 @@ def test_string_interpolation_with_inner_double_double_quotes(self):
         template = airspeed.Template('#set($d = "{""a"": 2}")$d')
         self.assertEqual('{"a": 2}', template.merge({}))
 
+    def test_string_interpolation_with_multiple_double_quotes(self):
```

Comment:
is this coming from a real example? because i did some testing/experimenting with the mapping tool and aws checks, and I don't think this would work in both scenarios. Anyway just checking

Reply:
Great catch - updated the sample, based on some more parity testing against AWS. 👍